### PR TITLE
Add support for various Parquet serde class names

### LINKED ISSUE #2518: TestHiveClient.testTypesParquet() is broken in cdh4 tests
The commit that broke it is acfe352844e9937e9b90e636032330614e349c20

### PR DESCRIPTION
**Pull request by dain**
Fixes #2518

### CODE REVIEW

File: presto-hive/src/main/java/com/facebook/presto/hive/ParquetRecordCursorProvider.java [hunk 1]
```diff
@@ -49,7 +55,7 @@
             DateTimeZone hiveStorageTimeZone,
             TypeManager typeManager)
     {
-        if (!isDeserializerClass(schema, ParquetHiveSerDe.class)) {
+        if (!PARQUET_SERDE_CLASS_NAMES.contains(getDeserializerClassName(schema))) {
```
Comment (cberner) [inline presto-hive/src/main/java/com/facebook/presto/hive/ParquetRecordCursorProvider.java]: Should this check be based on the `getSimpleName()`, or are those the only two packages that will contain one of these?

**Comment by cberner** [comment]
looks good, but see my comment

**Comment by dain** [comment]
I'll land this so the build passes.

I could go either way on using simple name.  @electrum do you have an opinion?

**Comment by electrum** [comment]
This seems fine. We can change it if there end up being more.

**Pull request merged**